src/construct_realm.js

```diff
@@ -17,9 +17,19 @@ import * as evaluators from "./evaluators/index.js";
 import * as partialEvaluators from "./partial-evaluators/index.js";
 import { NewGlobalEnvironment } from "./methods/index.js";
 import { ObjectValue } from "./values/index.js";
+import { Debugger } from "./debugger/Debugger.js";
+const fs = require("fs");
```

Comment:
You want to import this and you should also update package.json to make this module a runtime dependency.